keps/sig-network/3453-minimize-iptables-restore/README.md

```diff
@@ -498,7 +498,8 @@ modes.
 - 2022-10-04: Initial KEP merged
 - 2022-11-01: Code PR merged
 - 2022-11-10: Testing PRs merged; 5000Nodes test begins enabling the feature
-- 2022-12-08: Kubernetes 1.26 released
+- 2022-12-08: Alpha in Kubernetes 1.26
```

Comment:
Sorry, can't comment on particular lines, so commenting here:

L413: Have those metrics been added? If not, can we make it a GA graduation criteria? [and if they were, can you update the text there?]

And that's my only PRR comment - once addressed this LGTM for GA.

Reply:
doh! I wrote the PR but never submitted it

Reply:
Looking forward for it (happy to review it).

Reply:
just don't modify the existing ones, append the new ones, there is a lot of tooling depending on them